modules/projections/src/main/scala/ch/epfl/bluebrain/nexus/sourcing/projections/ProjectionProgress.scala

```diff
@@ -103,6 +110,8 @@ object ProjectionProgress {
 
     def minProgress: SingleProgress = this
 
+    def minProgressFilter(f: String => Boolean): SingleProgress = minProgress
```

Comment:
In this case the filter is ignored and returns `this` regardless if the filter does not select the current progress. In the case of `OffsetsProgress` it allows having NoProgress as a fallback (if the filter does not select anything). Should these behave the same?

Reply:
this is correct for a `SingleProgress`, since there are no progressIds at all.